modules/nf-core/quilt/quilt/main.nf

```diff
@@ -0,0 +1,62 @@
+process QUILT_QUILT {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
Is this an appropriate label for the tool? It looks like it is multithreaded and I suspect requires more than 6Gb of RAM in a lot of circumstances.

Reply:
Ok, that's fine, just wanted to ask the question.